Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM voidlock/erlang:R16B03-1-onbuild
+FROM voidlock/erlang:17.5-onbuild
```

Comment:
Any reason this is not using an official or public (and thus impersonal) docker image?